### PR TITLE
Updating the security disclosure process

### PR DESCRIPTION
**Pull request by michmike**

### CODE REVIEW

**Comment by michmike** [comment]
Original issue request - https://github.com/goharbor/harbor/issues/7501

**Comment by michmike** [comment]
Project Envoy Security Disclosure was our inspiration for the Harbor security disclosure process. we definitely captured a lot of their best practices and content.

**Comment by michmike** [comment]
@goharbor/all-maintainers  please review

File: SECURITY.md [hunk 1]
```diff
@@ -1,78 +1,90 @@
...
 * Description of the effects of the vulnerability on Harbor and the related hardware and software configurations, so that the Harbor Security Team can reproduce it.
 * How the vulnerability affects Harbor usage and an estimation of the attack surface, if there is one.
-* List other projects or dependencies that were used in conjunction with Harbor to produce the vulnerability..
- 
- 
-#Patch, Release, and Disclosure
+* List other projects or dependencies that were used in conjunction with Harbor to produce the vulnerability.
  
+## When to report a vulnerability
+* When you think Harbor has a potential security vulnerability
```
Comment (xaleeks) [inline SECURITY.md]: missing a period

File: SECURITY.md [hunk 2]
```diff
@@ -1,78 +1,90 @@
...
+* When you suspect a potential vulnerability but you are unsure that it impacts Harbor
```
Comment (xaleeks) [inline SECURITY.md]: missing a period

File: SECURITY.md [hunk 3]
```diff
@@ -1,78 +1,90 @@
...
-5.  The Security Team works on fixing the vulnerability and performs internal testing before preparing to roll out the fix.
-6.  The Security Team will first email the fix to cncf-harbor-distributors-announce@lists.cncf.io, so that they can further test the fix and gather feedback. See the section ‘Disclosure to Private Distributors List’ for details about how to join this mailing list.
-7.  Once the fix is confirmed, the Security Team will patch the vulnerability in the upcoming minor release and the next major release, and backport it into all earlier supported releases. In special cases, the fixes are cherry-picked into older unsupported releases for customers who cannot upgrade to a patched release.
-8.  The Security Team publishes an advisory to the Harbor community via Github, Slack, blog, and assists in rolling out the patched release to affected users.  
+3.  The Security Team will initiate a conversation with the reporter within 3 business days
```
Comment (xaleeks) [inline SECURITY.md]: missing a period

File: SECURITY.md [hunk 4]
```diff
@@ -1,78 +1,90 @@
...
+
+### Fix Disclosure Process
+The Security Team publishes an [advisory](https://github.com/goharbor/harbor/security/advisories) to the Harbor community via GitHub. In most cases, additional communication via Slack, Twitter, CNCF lists, blog and other channels will assist in educating Harbor users and rolling out the patched release to affected users. 
+
+The Security Team will also publish any mitigating steps users can take until the fix can be applied to their Harbor instances.
  
-#Disclosure to Private Distributors List
+## Mailing lists
+- Use cncf-harbor-security@lists.cncf.io to report security concerns. Received by Product Security Team members, and used by the security team to discuss security issues and fixes.
+- Join cncf-harbor-distributors-announce@lists.cncf.io for early private information, including mitigating steps, on security patch releases. See below for information on how Harbor distributors can apply to join this list.
```
Comment (xaleeks) [inline SECURITY.md]: it should say 'on how Harbor users can apply to join' right?

File: SECURITY.md [hunk 5]
```diff
@@ -1,78 +1,90 @@
...
+8.  Once the fix is confirmed, the Security Team will patch the vulnerability in the next patch or minor release, and backport a patch release into all earlier supported releases. 
+
+### Fix Disclosure Process
+The Security Team publishes an [advisory](https://github.com/goharbor/harbor/security/advisories) to the Harbor community via GitHub. In most cases, additional communication via Slack, Twitter, CNCF lists, blog and other channels will assist in educating Harbor users and rolling out the patched release to affected users. 
+
+The Security Team will also publish any mitigating steps users can take until the fix can be applied to their Harbor instances.
  
-#Disclosure to Private Distributors List
+## Mailing lists
+- Use cncf-harbor-security@lists.cncf.io to report security concerns. Received by Product Security Team members, and used by the security team to discuss security issues and fixes.
```
Comment (xaleeks) [inline SECURITY.md]: 'security team' should be capitalized

**Comment by xaleeks** [comment]
a few punctuation corrections, otherwise LGTM

**Comment by michmike** [comment]
> a few punctuation corrections, otherwise LGTM

made the changes. thanks Alex

**Pull request merged**